Automattic/jetpack
# Block performance tests: fix jetpack-connect script

#### Changes proposed in this Pull Request:
Looks like with #21848 the npm jetpack-connect script was missed, causing block performance tests workflow to fail.
I also fixed the wrong `tunnel-off` command in causing the environment teardown step to return non 0 exit code.

#### Jetpack product discussion
1156386383419466-as-1201474418952825

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Cannot be tested before merging with the current workflow triggers. I tested it by changing the trigger, see [this run](https://github.com/Automattic/jetpack/runs/4433315514?check_suite_focus=true)
